### ISSUE TITLE
Accessibility: MAS42A: High Contrast: No borders for the buttons/ header elements in high contrast black/white themes

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#18]
#A11yMAS
#MAS42A
#Wipro
#vscode-website
#Firefox

**Prerequisite:**
**_1. Change system setting to High contrast White_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast White" from the "Choose a theme" dropdown. Click on Apply.
**_2. Change system setting to High contrast Black_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast Black" from the "Choose a theme" dropdown. Click on Apply. 

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Repro Steps:**
**_Scenario-1:_**
1. Open [https://code.visualstudio.com/docs ](https://code.visualstudio.com/docs )in edge.
2. Navigate to the "Was this documentation helpful?" above the footer of the page.
3. Observe the borders for the "Yes","No" buttons  in high contrast black/white theme.

Note: Same issue observed in below links:
[https://code.visualstudio.com/updates](https://code.visualstudio.com/updates)
https://code.visualstudio.com/docs/supporting/faq
https://code.visualstudio.com/docs/introvideos/overview
https://code.visualstudio.com/docs/editor/accessibility
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/customization/userandworkspace
https://code.visualstudio.com/docs/languages/overview
https://code.visualstudio.com/docs/setup/setup-overview
https://code.visualstudio.com/docs/runtimes/nodejs

**_Scenario-2:_**
1. Open https://marketplace.visualstudio.com/VSCode in edge.
2. Navigate to the header elements of the page at top of the page and links under "Filter by category / collection" at above the footer of the page.
3. Observe the borders for the header elaments and links in high contrast contrast black/white . 

**Actual Result:**
**_Scenario-1:_**
The borders are not visible for the "Yes","No" buttons Below the "Was this documentation helpful?"  in high contrast black/white .
Note: User may not click the button and check ,as they are visible like normal text.

**_Scenario-2:_**
The borders are not properly visible for the header elements at top of the page and links under the "Filter by category / collection"  in high contrast contrast black/white . 

**Expected Result:**
**_Scenario-1:_**
There should be borders for the "Yes","No" buttons Below the "Was this documentation helpful?"  in high contrast black/white ,so that user can identify that they are buttons.

**_Scenario-2:_**
Proper borders should be visible for the header elements at top of the page and links under the "Filter by category / collection"  in high contrast contrast black/white . 

**User Impact:**
The persons with colour blindness and low vision will not be able to understand that they are buttons  and may not use the application effeciently.

**Screenshots:**
[Sc 1 No borders for the buttons.zip](https://github.com/wiproaccess/vscode-website/files/687145/Sc.1.No.borders.for.the.buttons.zip)
[Sc 2 No proper borders  for header elements and links.zip](https://github.com/wiproaccess/vscode-website/files/687146/Sc.2.No.proper.borders.for.header.elements.and.links.zip)

**Comment by waderyan** [comment]
Fixed for scenario 1 w/ e955edaeba831f29a04d4dd356bbebe8102a35c9

Other scenario is for the Marketplace team.

**Comment by wiproaccess** [comment]
The issue is verified hence adding the label 'closed' and closing the bug.
